repo.go

```diff
@@ -22,22 +22,20 @@ type Interface interface {
 	// GetBundle returns the bundle referenced by curl.
 	GetBundle(curl *charm.URL) (charm.Bundle, error)
 
-	// Latest returns the latest revision of the charms referenced by curls,
-	// regardless of the revision set on each curl.
-	Latest(curls ...*charm.URL) ([]CharmRevision, error)
-
 	// Resolve resolves the series and revision of the given entity
 	// reference. If the series is not specified, it may be resolved
 	// by the charm store or rejected. After the series is resolved,
```

Comment:
Comment adjusted. It's always ok to return an empty list of supported series, if the charm doesn't declare them (old charms).